doc/Rules.md

```diff
@@ -311,7 +311,7 @@ Corrected Example:
 ```
 
 Note: This rule applies to a sublist only if its parent lists are all also
-unordered (otherwise, extra indentation of ordered lists interferes with the
+unordered (otherwise, an extra indentation of ordered lists interferes with the
```

Comment:
In this context, "indentation" is uncountable, so adding the "an" is not warranted.